docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md

```diff
@@ -17,6 +17,21 @@ This section has instructions for configuring a pipeline for shipping logs avail
 * This solution supports only log files from Blob storage that have file extensions of .csv, .json, .blob, or .log. 
 * You also need to have Microsoft Authorization/role Assignments/write permissions, so they should be a "User Access Administrator" or "Owner".
 
+Watch this tutorial to learn how to collect logs from Azure Blob Storage.
+
+<Iframe url="https://www.youtube.com/embed/186K2d-FFoc?rel=0"
+        width="854px"
+        height="480px"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+import Iframe from 'react-iframe';
```

Comment:
Can we move this import Iframe command to top of the page and keep it with the import usebaseUrl.

Reply:
Good suggestion, @JV0812! Done.